AppInstaller/gradle/libs.versions.toml

```diff
@@ -0,0 +1,46 @@
+[versions]
+androidx-core = "1.6.0"
+appcompat = "1.3.1"
+material = "1.4.0"
+compose = "1.0.1"
+lifecycle = "2.4.0-rc01"
+activity = "1.3.1"
+navigation = "2.4.0-alpha05"
+protobuf = "3.10.0"
+datastore = "1.0.0"
+dagger = "2.38.1"
+hilt = "1.0.0"
+workmanager = "2.7.0-alpha05"
```

Comment:
2.7.1 seems to be the latest stable one https://developer.android.com/jetpack/androidx/releases/work